src/net/rpc.rs

```diff
@@ -1512,37 +1511,66 @@ impl ConversationHttp {
     /// Load up the canonical Stacks chain tip.  Note that this is subject to both burn chain block
     /// Stacks block availability -- different nodes with different partial replicas of the Stacks chain state
     /// will return different values here.
-    /// tip_opt is given by the HTTP request as the optional query parameter for the chain tip
-    /// hash.  It will be None if there was no paramter given.
+    ///
+    /// # Inputs
+    /// - `tip_opt` is given by the HTTP request as the optional query parameter for the chain tip
+    /// hash.  It will be None if there was no parameter given.
+    /// - `use_latest_tip` is also an optional query parameter, and defaults to false.
+    ///
     /// The order of chain tips this method prefers is as follows:
-    /// * tip_opt, if it's Some(..),
-    /// * the unconfirmed canonical stacks chain tip, if initialized
-    /// * the confirmed canonical stacks chain tip
+    /// * If `use_latest_tip` is true, the method attempts to return the unconfirmed chain tip,
+    ///     and returns the confirmed stacks chain tip if that doesn't work.
+    /// * If `tip_opt` is Some(tip), the method returns tip.
+    /// * The last option the function considers is the canonical Stacks chain tip.
```

Comment:
This is kind of confusing, how about "Otherwise, the function considers the canonical Stacks chain tip."